.travis.yml

```diff
@@ -89,9 +89,11 @@ jobs:
       script:
         - go get golang.org/x/tools/cmd/cover
         - go get github.com/mattn/goveralls
+        - go get -u github.com/ory/go-acc
```

Comment:
On the github.com/ory/go-acc project it says that it is a cross platform replacement of a bash script. Since the code coverage takes into account only Linux tests can we simply use that bash script instead of go-acc project?

Reply:
We could, but we are not adding the library in the repository, so I don't see the problem.

Reply:
Fine